src/data/ACTIONS/root/SCH.ts

```diff
@@ -6,6 +6,31 @@ export const SCH = ensureActions({
 	// Player
 	// -----
 
+	AETHERFLOW: {
+		id: 166,
+		name: 'Aetherflow',
+		icon: 'https://xivapi.com/i/000000/000510.png',
+		cooldown: 60000,
+	},
+
+	RESURRECTION: {
+		id: 173,
+		name: 'Resurrection',
+		icon: 'https://xivapi.com/i/000000/000511.png',
+		onGcd: true,
+		speedAttribute: Attribute.SPELL_SPEED,
+		castTime: 8000,
+	},
+
+	SCH_PHYSICK: {
```

Comment:
Does SCH code handle this rename?

Reply:
I tested one SCH log that had a SCH Physick and it seemed ok.  I'll look for a few more when I'm testing again tonight for further verification.